components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -2895,6 +2895,17 @@ paths:
       summary: Add a new API Key Manager
       description: |
         Add a new API Key Manager
+      parameters:
+      - name: tokenType
+        in: query
+        description: |
+          The type of the tokens to be used (exchanged or without exchanged)
+        schema:
+          type: string
+          enum:
+            - EXCHANGED
+            - ORIGINAL
+          default: ORIGINAL
```

Comment:
Add to DTO without using parameter

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/10567/commits/e1f31cb54fdf865a9fefa95e4ce9dd8bda815842